### ISSUE TITLE
[Story a11y] Open page attachments should make the content in the story not readable

### ISSUE THREAD

**Issue by mszylkowski** [ampproject/amphtml#38433]
### Description

When a page attachment is opened, the page attachment button in the page is readable by VoiceOver (even though the entire page is hidden behind the page attachment). The system layer buttons are also readable but disappear.

### Reproduction Steps

Open a story with VoiceOver that has a page attachment (eg: https://wsdemos.uc.r.appspot.com/cta-buttons with the "Explore Tasting Menu") and open a page attachment.

The expected behavior is that the page attachment should not be readable, but it is. The content of the page is readable and interferes with the page attachment reading flow. The system layer is also readable but disappears.

The accessibility tree in Google Chrome can be used to see that the page attachment UI in the page is part of the a11y tree even when hidden by its own page attachment.

Original story reported: https://stories.mlb.com/live/662000.html

### OS(s) Affected

All

### AMP Version Affected

latest
